zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedDiskChecker.scala

```diff
@@ -40,7 +40,7 @@ object DeletedDiskChecker {
                     ee.getCause.getMessage.contains("Compute Engine API has not been used") || ee.getCause.getMessage
                       .contains("This API method requires billing to be enabled")
                   )
-                    if (isDryRun) F.unit else dbReader.updateDiskStatus(disk.id)
+                    logger.info(s"failed to get ${disk.diskName} due to ${ee.getCause.getMessage}")
```

Comment:
Maybe I am missing something, but from your PR description it sounded like you wanted to keep the updateDisk behavior when getting the `"Compute Engine API has not been used"` message? It looks like the disk won't be updated for that case either though

Reply:
Sorry, I hadn't updated the description. After talking with @Qi77Qi . It sounds like updating shouldn't happen for either message.

This seems consistent with the conversation I found in an old PR relating to this:
https://github.com/broadinstitute/leonardo-cron-jobs/pull/320

Reply:
Gotcha, thanks for updating the descritption. In this case ping me when you have the logging piece and I can 👍 :)

Reply:
The logging and metrics are all updated. Thanks for your review.

Reply:
fwiw, I don't think cron jobs logs are in grafana..think they were previously in GCP, and we never ported them over to grafana.